src/test/java/AddSong.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.interactions.Actions;
+
+public class AddSong extends BaseTest {
+    @Test
+    public void addSongToPlaylist() {
+        enterEmail("rumkostia92@gmail.com");
+        enterPassword("te$t$tudent");
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/215854094-4e7d32c8-3886-42b1-9d58-a2a637254494.png)